src/components/number-picker/number-picker.component.ts

```diff
@@ -102,7 +102,7 @@ export class NumberPickerComponent implements ControlValueAccessor, OnDestroy {
     /** Store the current valid state */
     _valid: boolean = true;
 
-    /* */
+    /** Setting the isDestroy variable */
```

Comment:
Could we make this more descriptive about why it is needed, eg:

```typescript
/**
  * Determine if the component has been destroyed to avoid change detection being after the component
  * is no longer instantiated. A workaround for Angular Forms bug (https://portal.digitalsafe.net/browse/EL-369)
  */
```